tokio-util/Cargo.toml

```diff
@@ -7,7 +7,7 @@ name = "tokio-util"
 #   - Cargo.toml
 # - Update CHANGELOG.md.
 # - Create "v0.2.x" git tag.
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
It may be preferable to split this (bump crate version) into separate PRs (but personally either is fine).

Reply:
Mm, we will need a release preparation PR in any case.